### ISSUE TITLE
GEM sceneassetimporter.setreg .DAE is not on by default or in the ReadME on adding this mesh format support.

### ISSUE THREAD

**Issue by shawstar** [o3de/o3de#14124]
**Describe the bug**
GEM sceneassetimporter.setreg .DAE is not on by default or in the ReadME on adding this mesh format support.

This information should be in the readme or this file added to the gem Registry.

![image](https://user-images.githubusercontent.com/87207603/213491156-cc3f6934-073c-4255-998f-7159663cfc29.png)

**Steps to reproduce**
1. Add ROS2 Gem
2. Run with O3DE
3. URDF Meshes will not be process as .DAE is not listed for AP.

**Expected behavior**
.DAE meshes to be process and show up when URDF is imported.

**Actual behavior**
.DAE meshes will not show up when URDF is processed.

**Found in Branch**
Linux Nighty Development 1/18/2023

**Commit ID from [o3de/o3de](https://github.com/o3de/o3de) Repository**
Please provide the SHA/hash that identifies the latest commit from the o3de/o3de repository if the issue is reproducible in the development or other official branches. 
You can get the commit ID by running the `git rev-parse HEAD` command on your current branch.

**Desktop/Device (please complete the following information):**
 - Device: [Linux PC] 
 - OS: [Ubuntu ]
 - Version 22.04LT]
 - CPU [e.g. Intel I9-9900k ]
 - GPU [NVidia RTX 3090]
 - Memory [e.g. 16GB]

**Additional context**

**Comment by AMZN-stankowi** [comment]
Triage: This is a feature request (enable Collada importing), not a bug.

**Comment by lmbr-pip** [comment]
Accepting new file formats will require sign off from SIG/content. We support collada as an experimental format. It can be turned on to be a "default" but the expectation for bug fixing must be lower unless we have resources willing to tackle the collada specific issues.

**Comment by lgleim** [comment]
Discussed in today's Sig-Simulation triage. Referring to sig content for assessment.

**Comment by monroegm** [comment]
@o3de/sig-simulation @shawstar Is this a documentation request or is this about moving Collada support out of experimental?

**Comment by shawstar** [comment]
Yeah, I would move it our of experimental, its more functional than .glb and .gltf and they are enabled by default.  I have tested .dae static meshes, they work as you would expect. I have not tested them in an Animation workflow as yet however. This can be done if its whats stopping them out of experimental.

**Comment by shawstar** [comment]
Well, tested .DAE Animation, this is not working at the moment. The mesh will not render in O3DE.

**Comment by AMZN-stankowi** [comment]
Part of how we define if a format is in Experimental or not is automated tests: We have a lot of automated tests for FBX files. We have zero automated tests for GLB and GLTF, and zero for Collada.

If someone builds an RFC or engineering plan to move GLTF or Collada out of Experimental, it needs to include automated tests.

**Comment by hultonha** [comment]
Thanks for following-up @AMZN-stankowi. Do you know roughly how many there are currently for FBX and how much effort would be required to add such a suite of tests? Are we talking days/weeks/months? That would help us understand if/when this work might get done. Thanks!
